index.js

```diff
@@ -563,6 +580,11 @@ module.exports = function (log, indexesPath) {
       const seek = op.data.seek
       for (let i = 0, len = recs.length; i < len; ++i) {
         const { value, offset } = recs[i]
+        if (value === undefined) {
```

Comment:
I prefer `if (!value)` because it covers both `null` and `undefined` in case a null ever sneaks in. Or if you want to avoid all other falsy values, `if (value == null)` covers only `null` and `undefined`